package-lock.json

```diff
@@ -1,14130 +1,8 @@
 {
   "name": "israeli-bank-scrapers",
   "version": "1.0.4",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
If you didn't changed the `package.json` the `package.lock.json` should not be changed too.

I guess it is because you're using the wrong version of NodeJS, which needs to be `v16.19.0`.